test/services/messengerSync/rabbitmqMock.js

```diff
@@ -0,0 +1,17 @@
+const queues = {};
+
+const assertQueue = (queue, options) => {};
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['queue' is defined but never used. Allowed unused args must match /app|req|res|next|options|params/u.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5203703)